### ISSUE TITLE
compare field sampling dates

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/ospree#187]
Comparing: analyses/output/studytype_withBB.csv with dates2weeks (need to run analyses/limitingcues/studydesignplots_more.R to line 75)

Example of some differences:
ashby62 exp 1 -- I have no field.sample.dates, your file says 5 dates
ashby62 exp 2 -- I have 4 dates, your file say 5
biasi12 exp 1 -- only 9 dates two weeks apart?
Also, some give field.sample.date as 1 when I think there is no sample date (e.g., Basler12 exp1).

**Comment by cchambe12** [comment]
@lizzieinvancouver 

Ahh! Yes, my code was counting both blanks and NAs - I think it's all fixed now! As for biasi12, I just counted the number of field sample dates recorded in the ospree sheet, so 18 for the one species and 14 for the other. For the wein column, if there were 2 or more field sample dates that were at least 2 weeks apart then I gave it a 1. If they were within 2 weeks of each other, then I gave it a 0. Not sure if this is what we wanted...
